### ISSUE TITLE
Edit profile.ini.php to add [jdb:search] causes Error 500, PostgreSQL search

### ISSUE THREAD

**Issue by MidPiedmont** [3liz/lizmap-web-client#2470]
### Description
Attempting to get PostgreSQL search working on the web client.
Following Instructions from docs.lizmap.com, "Spatial Searching". 
Everything works in PG Admin, Materialized View `lizmap_search` is good to go.
`lizmap_search` is stored in public schema
Entering `SELECT * FROM lizmap_search LIMIT 1;` returns a result.

After adding in 
```ini
[jdb:search]
driver=pgsql
database="PostGIS"
host=http://(mywebdns).com
user=(myuser`)
password=(mypass)
; search_path=public,public
```
to `profiles.ini.php` with my db credentials obv, (mywebdns).com/lm gives me this Error:
`Error 500. A technical error has occured. Sorry for this trouble.`

If I remove the `[jdb:search]...` , (mywebdns).com/lm returns to normal.

This is my first post here. Hopefully, I'm following the correct conventions. If you need any more info or logs, just point me in that direction and I'll see what I can do. I thought about adding in logs, but I don't know what is relevant to my issue, or where to find them, other than in the web client and in server apache2 log.

### Environment
 - Lizmap version: **3.5** most recent, master, as of 1 week ago
 - QGIS Server FULL version: `1:3.20.2+34hirsute` **3.20.2**
 - QGIS Desktop FULL version: `1:3.20.2+34hirsute` **3.20.2**
 - OS: Server is **Ubuntu Desktop**, Client is **Windows**
 - Browser: **Chrome**
 - Lizmap admin information stored in: default **sqlite**

**Comment by MidPiedmont** [comment]
So I found a fix for the 500 error, since my lizmap/QGIS server and PostgreSQL are on the same machine, I just changed `host` from the DNS URL to `"localhost"`. Now Lizmap loads and the search bar shows up. 

It still should have worked the other way, especially considering that all of the other moving parts of my stack use the URL.

Now I am having another issue, the `lizmap_search` Materialized View is populated in the database, but I get no results when searching in the search bar.
